.laminas-ci.json

```diff
@@ -1,4 +0,0 @@
-{

```

Comment:
Seems like a negative diff on `.laminas-ci.json` completely breaks the action

Reply:
yes, my PRs were created from a script and this was a result of a bug. have to fix this manually